internal/controller/streams.go

```diff
@@ -389,11 +394,7 @@ func (sm *streamManager) StreamFalcoNetworkFlows(ctx context.Context, logger *za
 				logger.Error("Failed to parse Falco event into flow", zap.Error(err))
 				return err
 			}
-			err = sm.sendNetworkFlowRequest(logger, convertedFalcoFlow)
-			if err != nil {
-				logger.Error("Failed to send Falco flow", zap.Error(err))
-				return err
-			}
+			sm.flowChannel <- convertedFalcoFlow
 		}
```

Comment:
With my proposal, this now must read from the cache's `outFlows` and actually send those flows.

One way to do it is here, in the same `select`:
```suggestion
		case falcoFlow <-c.flowCache.outFlows:
			err = sm.sendNetworkFlowRequest(logger, falcoFlow)
			if err != nil {
				logger.Error("Failed to send Falco flow", zap.Error(err))
				return err
			}
		}
```
But that won't work for Cilium, since the body of the `for` loop in `exportCiliumFlows` is not one `select` like here.

So what I would suggest instead is that `streamManager` starts a separate goroutine to read flows from `outFlows` and send them. That would work for both Cilium and Falco.

Reply:
In order for streamManager to have access to that chan shall it also now include the flowcache object?